packages/reference/src/common/MultipleReferenceEditor.tsx

```diff
@@ -30,13 +30,13 @@ function onLinkOrCreate(
   entityType: ChildProps['entityType'],
   items: ChildProps['items'],
   ids: string[],
-  index?: number
+  index = items.length,
```

Comment:
Shouldn't the index be `length -1`?

Reply:
this is called before `items` is updated, so this works